session/pingpong/consumer_balance_tracker_test.go

```diff
@@ -255,10 +255,8 @@ func TestConsumerBalanceTracker_FallsBackToTransactorIfInProgress(t *testing.T)
 		bus: bus,
 	}
 	bc := mockConsumerBalanceChecker{
-		channelToReturn: client.ConsumerChannel{
-			Balance: initialBalance,
-			Settled: big.NewInt(0),
-		},
+		errToReturn:         errors.New("No contract deployed"),
```

Comment:
Why that change was made? It stabilizes test, but it seems it just hides the problem.

Reply:
If registration is in progress it means no channel SC is deployed in that address, so this call will fail

Reply:
If it succeded like it said it would mean registration is complete

Reply:
Ok, cleared